src/python/pants/help/maybe_color.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from colors import cyan, green, magenta, red
+
+
+class MaybeColor:
+    """A mixin to allow classes to optionally colorize their output."""
+
+    def __init__(self, color: bool) -> None:
```

Comment:
Would we ever want to make this color configurable? Emacs registers *faces* (see https://www.gnu.org/software/emacs/manual/html_node/elisp/Defining-Faces.html) which allow e.g. for the user to modify colors if their terminal doesn't interact well with the default. That seems like it would mostly just require turning this into a subsystem.

Reply:
Not a change I would expect in this PR.

Reply:
I could maybe see wanting that. But I think it would need to be a booststrap subsystem because Rust uses this option too. Right now, bootstrap options require a ton of ceremony and must be global options, so I think we'd want to improve that first.

We'd also need to stop using `ansicolors` to instead use another library (or honestly just us vendoring the relevant parts; it's a tiny amount of code). We probably need to do this soon anyways, as ansicolors hasn't been updated in some time and will break once 3.10 comes out afaict.